install_config/configuring_gce.adoc

```diff
@@ -182,6 +188,18 @@ ifdef::openshift-origin[]
 ----
 endif::[]
 
-To return to single-zone support, set the `multizone` value to `false`.
-
+Switching from not using a cloud provider to using a cloud provider produces an
+error message. Adding the cloud provider tries to delete the node because the
+node switches from using the *hostname* as the `*externalID*` (which would have
+been the case when no cloud provider was being used) to using the AWS
+`*instance-id*` (which is what the GCE cloud provider specifies). To resolve
+this issue:
 
+.  Log in to the CLI as a cluster administrator.
+.  Delete the nodes:
++
+----
+$ oc delete node <node_name>
+----
+.  On each node host, restart the *atomic-openshift-node* service.
```

Comment:
The same with AWS, for "atomic-openshift-node" looks just fit for Openshift Enterprise product.

Reply:
We have ifdefs in file. therefor for Openshift origin it will show `systemctl restart origin-node` and for Enterprise it will show `systemctl restart atomic-openshift-node`